sdcm/cluster_aws.py

```diff
@@ -255,6 +255,9 @@ def __init__(self, ec2_instance, ec2_service, credentials,
                           cluster.TEST_DURATION)
             self._ec2_service.create_tags(Resources=[self._instance.id],
                                           Tags=[{'Key': 'keep', 'Value': 'alive'}])
+        self._ec2_service.create_tags(Resources=[self._instance.id],
+                                      Tags=[{'Key': 'workspace', 'Value': cluster.WORKSPACE},
+                                            {'Key': 'uname', 'Value': ' | '.join(os.uname())}])
```

Comment:
Why call twice the create_tags method? Just add the new tags to the list.

Reply:
The first one is only called when `test duration` is larger than 1 day. But I want to create new tags all the time.

Reply:
Amos, my question was about not to call twice the same method, just add keys to the first call.

Reply:
OK, I saw it. will send a PR to improve this.